src/bundle/Resources/views/layout.html.twig

```diff
@@ -93,6 +93,14 @@
             'bundles/ezplatformadminuiassets/vendors/create-react-class/create-react-class.min.js'
             'bundles/ezplatformadminuiassets/vendors/prop-types/prop-types.min.js'
             'bundles/ezplatformadminuimodules/js/UniversalDiscovery.module.js'
+        %}
+            <script type="text/javascript" src="{{ asset_url }}"></script>
+        {% endjavascripts %}
+
+        {{ ezplatform_admin_ui_component_group('custom-admin-ui-modules') }}
+        {{ ezplatform_admin_ui_component_group('custom-admin-ui-config') }}
```

Comment:
and this is for adding config to the `eZ.adminUiConfig.universalDiscoveryWidget.extraTabs`?